Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -236,7 +236,7 @@ def initialize(preferenceFiles, frontend)
   end
 
   # Corresponds to WebFeatureStatus enum cases. "developer" and up require human-readable names.
-  STATUSES = %w{ embedder unstable internal developer testable preview stable }
+  STATUSES = %w{ embedder unstable internal developer testable preview stable shipping }
```

Comment:
We should add `WebFeatureStatusShipping` and `API::FeatureStatus::Shipping` cases, too. Right now, it's not a build failure, because `shipping` features aren't part of the experimental/internal-debug-only templates. Once #8345 lands, WebKit will generate code for `shipping` features as part of WebPreferences::features(), and we'll need those enum cases.

Reply:
Ah! Gotcha. I just updated with those bits added.

Reply:
Thank you!